app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> int:
```

Comment:
In general, this function code is so bad. Why don't you use get_rating() to get a rating? Rewrite it and use sum() and get_rating().